MyString.java

```diff
@@ -18,13 +18,40 @@ public static void main(String[] args) {
 
     /** Returns the lowercase version of the given string. */
     public static String lowerCase(String str) {
-        // Replace the following statement with your code
-        return null;
+        String lowerString = "";
+        for (int i = 0; i < str.length(); i++) {
+            char ch = str.charAt(i);
+            if (ch >= 'A' && ch <= 'Z') {
+                lowerString += (char) (ch + 32);
```

Comment:
+32 is not ideal ... it is unclear.. 
I rather you use `(char)(ch - 'A' + 'a')`